### PR TITLE
314 reviewing general rescheduling dialog

### PR DESCRIPTION
**Pull request by 7lopol7**
Fixes: https://github.com/PerfectFit-project/virtual-coach-issues/issues/314

preparation phase and rescheduling dialog do use a lot of the same actions now, so maybe look into how to best structure this.

### CODE REVIEW

File: Rasa_Bot/data/rules_rescheduling_dialog.yml [hunk 1]
```diff
@@ -1,35 +1,57 @@
...
+    - action: rescheduling_now_or_later_form
+    - active_loop: rescheduling_now_or_later_form
+    - slot_was_set:
+      - rescheduling_now: 1
+    - active_loop: null
+    - action: utter_user_chose_now
+    - action: action_set_continuation
+    - slot_was_set:
+      - dialog_to_continue: future_self_preparation
+    - action: action_continue_with_future_self_dialog_after_rescheduling_query
```
Comment (wbaccinelli) [inline Rasa_Bot/data/rules_rescheduling_dialog.yml:37]:
```suggestion
```

File: Rasa_Bot/data/stories_future_self_dialog.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ stories:
 
 - story: continue with future self dialog after rescheduling query
   steps:
-    - checkpoint: continue_with_future_self_dialog_after_rescheduling_query
+    - intent: EXTERNAL_trigger_continue_future_self_activity
```
Comment (wbaccinelli) [inline Rasa_Bot/data/stories_future_self_dialog.yml:24]: Also here, we don't need this. The whole future self dialog is to be deleted

File: Rasa_Bot/domain/domain_future_self_dialog.yml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 intents:
   - request_future_self_dialog
+  - EXTERNAL_trigger_continue_future_self_activity
```
Comment (wbaccinelli) [inline Rasa_Bot/domain/domain_future_self_dialog.yml:3]:
```suggestion
```

File: Rasa_Bot/domain/domain_preparation_phase.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ##Domain for preparation phase dialog
 intents:
-  - EXTERNAL_preparation_phase
+  - test_intent
```
Comment (wbaccinelli) [inline Rasa_Bot/domain/domain_preparation_phase.yml:3]: Let's use the final version.

**Review by wbaccinelli — APPROVED** [review]
Well done! One main comment is that we are not using the future self dialog, so the rescheduling is just for the general activity dialog

**Comment by 7lopol7** [comment]
@wbaccinelli I changed some things according to your suggestions, haven't been able to test it yet, because of some errors in my rasa server

**Comment by wbaccinelli** [comment]
Integrated in the resolution of https://github.com/PerfectFit-project/virtual-coach-issues/issues/338

**Pull request closed without merging**